README.md

```diff
@@ -101,3 +101,10 @@ $client->setPackagistUrl('https://custom.packagist.site.org');
 ## License
 
 `packagist-api` is licensed under the MIT License - see the LICENSE file for details.
+
+## Maintainers
+
+Please read [this post](https://knplabs.com/en/blog/news-for-our-foss-projects-maintenance) first.
+
+This library is maintained by the following people (alphabetically sorted) :
```

Comment:
Maybe also state that we're looking for maintainers, as robbieaverill prefers not to be the only person maintaining this project.

Reply:
@robbieaverill, is @nicolasmure proposal fine with you?

Reply:
Yes, of course :-)